docs/async.md

```diff
@@ -0,0 +1,83 @@
+# Async&lt;T&gt;
+Mavericks makes dealing with async requests like fetching from a network, database, or anything that can be mapped to an observable incredibly easy. Mavericks includes `Async<T>`. Async is a [sealed class](https://kotlinlang.org/docs/reference/sealed-classes.html) with four subclasses:
```

Comment:
should we still use the term `observable` when we are moving towards coroutines?